src/main/java/http_server/Request.java

```diff
@@ -0,0 +1,33 @@
+package http_server;
+
+import java.util.HashMap;
+
+public class Request {
+    private String method;
+    private String route;
+    private String body;
+    private HashMap<String, String> headers;
+
+    public Request(String method, String route, String body, HashMap<String, String> headers) {
```

Comment:
If this ever gets unwieldy, you could probably improve it using a factory or builder object.  I only mention this because with 4 arguments, growing this object might get tough over time for any consumer object using it.